htdocs/supplier_proposal/card.php

```diff
@@ -1692,7 +1692,8 @@
 		print '<table class="nobordernopadding" width="100%"><tr><td>';
 		print $langs->trans('DeliveryDate');
 		print '</td>';
-		if ($action != 'editdate_livraison' && $object->statut == SupplierProposal::STATUS_VALIDATED) {
+		// Pourquoi avoir une régle diffrente des autres champs pour l'edition de la date ?
```

Comment:
I see no reason for this test. Test on not signed as suggested seems better.

Can you remive the comment so i can validate pr. Question on code should be added in github comment, not into code itself.

Reply:
yes it's a change i notice for me and i forgot to remove